examples/csr/csr.c

```diff
@@ -166,9 +166,30 @@ int TPM2_CSR_ExampleArgs(void* userCtx, int argc, char *argv[])
         argc--;
     }
 
+    /* init wolfssl, required for fips */
+#ifdef WOLFCRYPT_ONLY
```

Comment:
This logic should be in `wolfTPM2_Init` and `wolfTPM2_Cleanup`? I am pretty sure these init/cleanups are already called. Perhaps the issue is I only call the wolfCrypt_Init version?

Reply:
Please swap out the logic you added in csr.c, but put into tpm2.c and add the `#ifdef WOLFCRYPT_ONLY` logic. Call wolfSSL_Init by default unless WOLFCRYPT_ONLY is defined.

Reply:
https://github.com/wolfSSL/wolfssl/blob/master/src/ssl.c#L6421
I think this is why, I'm going to call `wolfSSL_Init` when not `WOLFCRYPT_ONLY` and call `wolfCrypt_Init` and `wc_SetSeed_Cb` when it is